package.json

```diff
@@ -2,7 +2,9 @@
   "name": "react-use-event-hook",
   "version": "0.9.2",
   "description": "Same as React's `useCallback`, but returns a stable reference.",
-  "main": "dist/useEvent.js",
+  "main": "dist/index.js",
```

Comment:
Outputting to 2 sibling files (eg. `useEvent.js, useEvent.mjs`) does seem have some problems with some build tools.

There are some solutions listed here: https://www.sensedeep.com/blog/posts/2021/how-to-create-single-source-npm-module.html

TL;DR; I think it's best to output to 2 separate folders, eg. `dist/cjs/useEvent.js` and `dist/mjs/useEvent.mjs`

Reply:
See #10